packages/clay-css/src/scss/components/_empty-state.scss

```diff
@@ -63,3 +63,132 @@
 		@include clay-css(map-get($c-empty-state-sm, c-empty-state-footer));
 	}
 }
+
+// Empty State Animation
+.c-empty-state-animation {
+	.empty-state-animation {
+		@media (prefers-reduced-motion: no-preference) {
+			.ship {
+				transform-origin: bottom;
+				animation: 6s ease-in-out infinite ship;
```

Comment:
We need to alphabetize all CSS properties.

Reply:
animation needs to be delcared before transform-origin.

Reply:
Thanks, Pat! I had this fixed but did some rollback and forgot to sort it again.
I believe everything is ok now